update/Propagating_Updates.md

```diff
@@ -0,0 +1,66 @@
+# Propagating Updates
+
+A discussion on updating an existing WSO2 product deployment in a container platform.
+
+## Contents
+
+* [When do we need to update?](#when-do-we-need-to-update?)
+* [Simplest way to perform an update](#simplest-way-to-perform-an-update)
+* [Achieve zero downtime](#achieve-zero-downtime)
+
+### When do we need to update?
+
+The following are some of the scenarios which instigates the need for an update to an existing
+WSO2 product deployment in a container platform.
+
+* Integrate [WSO2 Updates](https://wso2.com/updates) containing an improvement or a bug fix
+* A version migration of the used WSO2 product
+* Updates to configuration files and non-configuration artifacts (e.g. binaries such as, third-party libraries,
+Carbon extensions in the form of OSGi bundles, [Carbon Applications](https://docs.wso2.com/display/Carbon440/Working+with+Carbon+Applications)
+or any security related artifacts such as, Java Keystore files)
+* Integrate changes to the container image source
```

Comment:
Include changes to the Container image source

Reply:
@chamindi-a I think we should stick to the verb `Integrate` here, as `Include` actually does not imply the meaning, i.e. to perform a change to container image source and deploy it.

Further, this is applicable to the next point which is about `installation resources` as well.

WDYT?

Reply:
@chirangaalwis: I have tried a change as the same verb was used in three instance. if we can changes this iam ok